src/client/testing/create-availability.spec.ts

```diff
@@ -3,71 +3,81 @@ import RootPage from './page-objects/root';
 import OAuthLoginPage from './page-objects/oauth';
 import SiteSelectionPage from './page-objects/site-selection';
 import SitePage from './page-objects/site';
+import DateUtils from './utils/date-utility';
 import CreateAvailabilityPage from './page-objects/create-availability';
+import SummaryPage from './page-objects/create-availability/summary-page';
 
 let rootPage: RootPage;
 let oAuthPage: OAuthLoginPage;
 let siteSelectionPage: SiteSelectionPage;
 let sitePage: SitePage;
 let createAvailabilityPage: CreateAvailabilityPage;
+let dateUtils: DateUtils;
+let summarypage: SummaryPage;
 
 test.beforeEach(async ({ page }) => {
   rootPage = new RootPage(page);
   oAuthPage = new OAuthLoginPage(page);
   siteSelectionPage = new SiteSelectionPage(page);
   sitePage = new SitePage(page);
   createAvailabilityPage = new CreateAvailabilityPage(page);
-});
+  dateUtils = new DateUtils();
```

Comment:
If we make this a module as suggested in my other comments, we won't need to new it up or declare it here